src/pages/hall-of-fame.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import data from '../assets/hall-of-fame.json';
+import HallOfFameTableComponent from '../components/hall-of-fame-table-component'
+const codechefData = data.codechefRankers;
+const codeforcesData = data.codeforcesRankers;
+
+export default function HallOfFame() {
+
+  return (
+    <div className='bg-green-700 '>
```

Comment:
Give this h-100v and add 100v as 100vh in tailwindconfig see [this](https://github.com/linustribevnr/turinghut-website/blob/main/src/pages/home.js#L6) and [this](https://github.com/linustribevnr/turinghut-website/blob/main/tailwind.config.js#L10)
This will make the component take full screen.

Reply:
Also add the primary and second colors and use those instead, we can change the colors later directly in the config file instead of dealing with the code later.